Shared/Samples/Create convex hull around points/CreateConvexHullAroundPointsView.swift

```diff
@@ -0,0 +1,125 @@
+            .onSingleTapGesture { _, mapPoint in
+                model.inputPoints.append(mapPoint)
+                model.pointsGraphicsOverlay.addGraphic(Graphic(geometry: mapPoint, symbol: model.markerSymbol))
+                model.createIsDisabled = false
+                model.resetIsDisabled = false
+            }
+            .toolbar {
+                // Create button for making the convex hull.
+                ToolbarItem(placement: .bottomBar) {
+                    Button("Create") {
+                        model.createConvexHull()
+                    }
+                    .disabled(model.createIsDisabled)
+                }
+                // Reset button for removing all the points on the map.
+                ToolbarItem(placement: .bottomBar) {
+                    Button("Reset") {
+                        model.inputPoints.removeAll()
+                        model.pointsGraphicsOverlay.removeAllGraphics()
+                        model.convexHullGraphicsOverlay.removeAllGraphics()
+                        model.createIsDisabled = true
+                        model.resetIsDisabled = true
+                    }
+                    .disabled(model.resetIsDisabled)
+                }
+            }
+    }
+}
+
+private extension CreateConvexHullAroundPointsView {
+    // The view model for this sample.
+    private class Model: ObservableObject {
+        /// A map with a topographic basemap.
+        var map: Map = {
+            let map = Map(basemapStyle: .arcGISTopographic)
+            return map
+        }()
+        
+        /// An Array of inputted points to be used in creating the convexHull.
+        var inputPoints: [Point] = []
+        
+        /// An Array that contains the graphics overlays for the sample.
+        lazy var graphicsOverlays: [GraphicsOverlay] = [
+            pointsGraphicsOverlay,
+            convexHullGraphicsOverlay
+        ]
+        
+        /// A GraphicsOverlay for the inputted points graphics.
+        var pointsGraphicsOverlay = GraphicsOverlay()
+        
+        /// A GraphicsOverlay for the convex hull graphic.
+        var convexHullGraphicsOverlay = GraphicsOverlay()
+        
+        /// A red simple marker symbol to display where the user tapped on the map.
+        let markerSymbol = SimpleMarkerSymbol(style: .circle, color: .red, size: 10)
+        
+        /// A blue simple line symbol for the outline of the convex hull graphic.
+        private let lineSymbol = SimpleLineSymbol(style: .solid, color: .blue, width: 4)
+        
+        /// A hollow polygon simple fill symbol for the convex hull graphic.
+        private lazy var fillSymbol = SimpleFillSymbol(style: .noFill, outline: lineSymbol)
+        
+        /// A Bool indicate whether the create button can be pressed.
+        @Published var createIsDisabled = true
+        
+        /// A Bool indicate whether the reset button can be pressed.
+        @Published var resetIsDisabled = true
```

Comment:
I tend to put these as state properties on the view as they are (almost) completely decided by the UI interactions - except that when the `if let` branch in `createConvexHull` isn't visited, which is 99.99% unlikely.

Also, looking back now I think it might be better to only use 1 Button and flip the text between "Create" and "Reset". See a similar example in `Cut geometry` sample.

Reply:
I think it might be better to keep the 2 button design just because the common design specifies both a Create and Reset button, so I changed it to a ToolbarItemGroup. Let me know if you want something different.

Reply:
Correct. 🤦 I was probably looking at sth else when I made this comment. 2 buttons work better as both of them can be in enabled state at the same time.